portal-web/test/functional/com/liferay/portalweb/macros/headless/objects/ObjectDefinitionAPI.macro

```diff
@@ -329,4 +348,19 @@ definition {
 		return "${getObjectsWithNestedFieldJSON}";
 	}
 
+	macro updateEmployee {
+		ObjectDefinitionAPI._updateEmployee(
+			employeeId = "${employeeId}",
+			firstname = "${employeeFirstname}",
+			newManagerId = "${newManagerId}");
+
+		var firstname = ObjectDefinitionAPI._getEmployeeFirstnameById(employeeId = "${employeeId}");
+
+		TestUtils.assertEquals(
+			actual = "${firstname}",
```

Comment:
Maybe we can assert here the code of the response, when editing something it should return code "204". I'm not sure if this information is passed in the response body, but maybe it's worth checking, as we may need it in the future.

Reply:
@magjed4289 How can I achieve return code 204? It does not appear in response

Reply:
@magjed4289 How can I achieve return code 204? It does not appear in response

Reply:
thanks for checking it @Hijky! Like we've talked about in the meeting, in this case, let's just assert that in the JSON response we get the same employeeId that we've asked for. And I will write down a request to the Poshi team about it in the future. thanks! :)

Reply:
sorry these comments, don't make sense now. I've just noticed the method we're using it in. Reviewing it again!